accord-core/src/main/java/accord/impl/SimpleProgressLog.java

```diff
@@ -277,7 +277,7 @@ void run(SafeCommandStore safeStore, SafeCommand safeCommand)
 
                                                 updateMax(success.asProgressToken());
                                             }
-                                        });
+                                        }).begin(node.agent());
```

Comment:
the leak detector found this case.  `CommandStore` returns a `AsyncChain` and not `AsyncResult`!  In Cassandra we use `AsyncChains.Head` so require `begin` to start, but in the in-memory stories we returned a `AsyncResult` which would make code work under tests but not in C*...